ui/de-lib/src/main/java/org/iplantc/de/apps/integration/client/view/propertyEditors/widgets/ArgumentValidatorEditor.java

```diff
@@ -338,4 +338,10 @@ void onEditButtonSelected(@SuppressWarnings("unused") SelectEvent event) {
         dlg.show();
     }
 
+    List<ArgumentValidator> getValidators() {
+        List<ArgumentValidator> values = Lists.newArrayList();
+        values.addAll(validatorStore.getAll());
+        return values;
+    }
```

Comment:
Yeah, it's specifically because I'm modifying the model.  If I leave it as an unmodifiable list, the next time the editor gets flushed, an error gets thrown because the ListStoreEditor will try to run
`model.clear();`
`model.addAll(store.getAll());`
and then error out.